article.js

```diff
@@ -44,3 +44,15 @@ setCurrentEtag = function () {
     }
   });
 };
+
+CompleteArticle.findByCategory = function(category, callback) {
+  webDB.execute(
+    [{
+      'sql': 'SELECT * FROM articles WHERE category = ?',
+      'data': [category]
+    }],
+    function (){
```

Comment:
What is this function all about? You want to use the callback below here, not an empty function.